### DISCUSSION TITLE
Release 1.0.0 Candidate

### DISCUSSION THREAD

**Discussion by StephenOman** [heiner/nle#11] [General]
With the change from OpenAI Gym to Farama Foundation's Gymnasium, we're suggesting the next release be 1.0.0. Changing the major version is usual when introducing breaking changes, which this change does.

Should this be done now or should we wait until the NetHack code is patched to 3.6.7?

**Comment by StephenOman** [comment]
NetHack is patched to 3.6.7, so I suggest that we release the current version as 1.0.0 to PyPI. Then it's about getting it into people's hands to get some feedback on how it's working for them.

**Comment by heiner** [comment]
SGTM.

**Comment by heiner** [comment]
If https://github.com/heiner/nle/pull/19 works at all we might include that as it sounds useful (although I'd like to get my asks for it resolved first).
